docs/tutorials/first-program.md

```diff
@@ -108,17 +109,16 @@ Now that we've added the function that will convert the incoming message text to
 
 This completes the sequence. If you go to the Source view, your program should now look like this:
 
-```
+```Ballerina
  import ballerina.net.http;
+
 @http:BasePath{value:"/myecho"} 
 service myEchoService {
-	@http:POST {} 
-	resource myEchoResource( message m) {
-		http:convertToResponse( m );
-		reply m;
-		
-	}
-	
+    @http:POST {} 
+    resource myEchoResource( message m) {
+        http:convertToResponse( m );
```

Comment:
Shall we move to the new API as message, http:convertToResponse(m) and reply m are no longer availabe?